Lib/inspect.py

```diff
@@ -3352,7 +3352,11 @@ def __str__(self):
             # flag was not reset to 'False'
             result.append('/')
 
-        rendered = '({})'.format(', '.join(result))
+        params = ', '.join(result)
+        if pretty and len(params) > 78:  # 80 - '(' - ')'
```

Comment:
pydoc will also display the function name on the same line.

Reply:
Indeed!